astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -2,16 +2,65 @@
 
 """Test geodetic representations"""
 import pytest
-from numpy.testing import assert_array_equal
 
+from numpy.testing import assert_array_equal
 from astropy import units as u
+from astropy.units import allclose as quantity_allclose
```

Comment:
Are you using `pre-commit`? I think this resort will not survive it!

Reply:
Not sure I understand this comment.
The change was another mistake on my side: I've reverted and put the line where it was at the beginning.
I am indeed using pre-commit and it did not complain.
I have synchronized with upstream main (and rebased), just in case the pre-commit configuration had changed.

Reply:
...now that I'm fighting with ruff I understand the comment...